actors/builtin/reward/reward_actor.go

```diff
@@ -65,7 +68,8 @@ func (a Actor) AwardBlockReward(rt vmr.Runtime, params *AwardBlockRewardParams)
 	penalty := abi.NewTokenAmount(0)
 	var st State
 	rt.State().Readonly(&st)
-	blockReward := big.Div(st.ThisEpochReward, big.NewInt(builtin.ExpectedLeadersPerEpoch))
+
+	blockReward := st.ThisEpochReward
```

Comment:
Blocking: a number of other places depend on this value being the total reward (expectation), not the per miner value. If we change this, those call sites will all have to multiply (or have the `ThisEpochReward` method do it for them).

If this was an optimization to divide once instead of 5 times, the net effect will be worse, since many more than 5 multiplies will be needed to make up for it.

Reply:
Mul is an order (or two) of magnitude cheaper than Div but alright.

Reply:
We could store both, but I think we'll only need 5 divs per epoch, which is hardly worth optimizing.